people/time-off.md

```diff
@@ -44,6 +44,25 @@ Follow the process described above in this case - you are welcome to describe yo
 In this case, our guidelines around letting others know ahead of time are somewhat relaxed, as sick leave may happen unexpectedly.
 Just do the best you can, and try to balance your own constraints with respect for the team’s bandwidth and planning.
 
+(time-off:annual-expected)=
+## End-of-year expected time off
+
+The **final 14 days of the year are "expected time off" for all of 2i2c**.
+
+During this time our organization operates at a reduced capacity and with greatly diminished commitments to our {external+docs:ref}`service level objectives <objectives:stability>`.
+
+Our support and operational roles are still staffed, but with greatly reduced expectations around responsiveness and resolution.
+Team members can use this time as they wish (including continuing to work), but we wish to set the _expectation_ with internal and external partners that we will be less responsive.
+
+Here is our policy during this time:
```

Comment:
I really like this list setting clear expectations for ourselves and external parties.

Reply:
I like this list